beta-3-setup.md

```diff
@@ -3179,6 +3186,26 @@ wildcard space:
     foo.cloudapps.example.com 0 IN A 192.168.133.2
     ...
 
+### Resolving From Your Local Machine
+If you would like to use another system on the network to view and the OSE console you will need to
+ensure the following:
+
+#### Hosts
+Edit ```/etc/hosts``` adding a line pointing to your ```ose3-master``` machine.
+
+Example:
+
+    127.0.0.1       localhost.localdomain localhost
+    ::1     localhost6.localdomain6 localhost6
+    192.168.122.86 ose3-master.example.com
+
+
+#### Resolv
+Edit ```/etc/resolv.conf``` and add your ```ose3-master``` machine as the first nameserver.
+
+**Note**: If you are using NetworkManager your ```/etc/resolv.conf``` will change over time. If you find this an issue you can make the file immutable ```chattr +i /etc/resolv.conf```. Do not forget to run ```chattr -i /etc/resolv.conf``` after you are finished with these labs else NetworkManager won't be able to modify the file!
```

Comment:
Wouldn't it be preferable to modify the dhclient configuration?

Also, adding "ose3-master" as the first nameserver would result in pointing at SkyDNS, which doesn't have DNS entries for the routes. If anything, you would want to be pointing at whatever node you're running dnsmasq on?